lua/nvim-treesitter/parsers.lua

```diff
@@ -2044,10 +2044,11 @@ list.slang = {
 
 list.slim = {
   install_info = {
-    url = "https://gitlab.com/theoreichel/tree-sitter-slim",
+    url = "https://github.com/theoo/tree-sitter-slim",
```

Comment:
(It's fine to point to your gitlab repo here, by the way; it's just about the maintainers entry which needs to be a user on Github -- the "owner" is clear from the repo URL.)